api/client/stack/remove.go

```diff
@@ -63,6 +63,11 @@ func runRemove(dockerCli *client.DockerCli, opts removeOptions) error {
 		}
 	}
 
+	if len(services) == 0 && len(networks) == 0 {
```

Comment:
@allencloud i might be missing something but what does this mean wrt a stack? As in, what's a stack with no services but only networks? just trying to make sense of this.